lib/opensusebasetest.pm

```diff
@@ -616,6 +616,9 @@ sub post_fail_hook {
     return if testapi::is_serial_terminal();    # unless VIRTIO_CONSOLE=0 nothing below make sense
 
     show_tasks_in_blocked_state;
+    # Find out in post-fail-hook if system is I/O-busy, poo#35877
+    my $io_status = script_output("sed -n 's/^.*da / /p' /proc/diskstats | cut -d' ' -f10");
+    record_info('System I/O status:', ($io_status =~ /^0$/) ? 'idle' : 'busy');
```

Comment:
Careful, we would now try to call `script_output` in the generic post_fail_hook which can fail basically *anywhere*, e.g. see L631 and below. Pretty sure that we should not try to call this when e.g. in "wait_boot" or "boot_desktop"?

Reply:
hmm that makes sense - but the only thing I can think about right now, is to introduce a variable which is set after wait_boot or boot_to_desktop are finished and use this variable as conditional trigger for a possible post_fail_hook

Reply:
updated now.